RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRegistry.java

```diff
@@ -1130,6 +1139,252 @@ private boolean parseSamPluginOutput(String regFilePath, AbstractFile regAbstrac
         return false;
     }
 
+    
+    /**
+     *  Finds the Host and Domain information from the registry.
+     */
+    private void parseSystemHostDomain() {
+        List<AbstractFile> systemFiles = new ArrayList<>();
+        org.sleuthkit.autopsy.casemodule.services.FileManager fileManager = currentCase.getServices().getFileManager();
+
+        // find the system hives', process this first so we can map the user id's and sids for later use
+        try {
+            systemFiles = fileManager.findFiles(dataSource, "system", "/system32/config"); //NON-NLS
```

Comment:
Is there a way we can re-use what is in findRegistryFiles()?

Reply:
Yes and done.